index.html

```diff
@@ -3277,31 +3277,58 @@ <h3>Reserved Extension Points</h3>
 see Section <a href="#types"></a>.
         </p>
 
-        <p class="issue" title="Reserved properties under debate">
-The following extension point properties are under consideration for being
-marked as reserved: `presentationSchema`, `credentialRefresh`, `termsOfUse`,
-`evidence`, `renderMethod`, `refreshService`, and
-`confirmationMethod/confidenceMethod`.
-        </p>
-
         <table class="simple">
           <thead>
             <tr>
               <th>Reserved&nbsp;Property</th>
               <th>Description</th>
             </tr>
           </thead>
-
           <tbody>
             <tr>
+              <td>`confirmationMethod` / `confidenceMethod`</td>
```

Comment:
I'll accept this PR as-is since there's an issue in here. Just expressing my preference:

```suggestion
              <td>`confidenceMethod`</td>
```

Reply:
We haven't agreed on either yet. I'd prefer to keep both until we resolved the naming discussion.